### PR TITLE
Automated OpenEBS version update 3.3.1

### PR DESCRIPTION
**Pull request by replicated-ci-kurl**
Automated changes by the [cron-openebs-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-openebs.yaml) GitHub action

```release-note
Adds [OpenEBS add-on](https://kurl.sh/docs/add-ons/openebs) version 3.3.1.
```

### CODE REVIEW

File: web/src/installers/versions.js [hunk 1]
```diff
@@ -355,6 +355,7 @@ module.exports.InstallerVersions = {
     "1.12.0",
     "1.6.0",
     // cron-openebs-update-3
+    "3.3.1",
```
Comment (rrpolanco) [inline web/src/installers/versions.js:358]: It's worth mentioning and clarifying that this is the version of the Helm chart NOT a new version of the openebs application.

**Comment by emosbaugh** [comment]
We should not release this update. There are no changes as only the chart version was updated. 

```
$ diff -r addons/openebs/3.3.0 addons/openebs/3.3.1/
diff -r addons/openebs/3.3.0/spec/openebs.tmpl.yaml addons/openebs/3.3.1/spec/openebs.tmpl.yaml
8c8
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
21c21
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
64c64
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
114c114
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
133c133
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
262c262
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
361c361
<     chart: openebs-3.3.0
---
>     chart: openebs-3.3.1
```

**Comment by rrpolanco** [comment]
Chart version `3.3.1` has updates to fix image tag versions: https://github.com/openebs/charts/commit/677ebb79e1efe672c9ca1fac7ad5e824e1a1dd1e

**Comment by rrpolanco** [comment]
> Testgrid Run(s) Executing @ https://testgrid.kurl.sh/run/pr-3415-0decfb7-openebs-3.3.1-k8s-docker-localpv-2022-09-15T19:43:48Z

`local pv upgrade from 2.6.0` [failed](https://testgrid.kurl.sh/run/pr-3415-0decfb7-openebs-3.3.1-k8s-docker-localpv-2022-09-15T19:43:48Z?kurlLogsInstanceId=phkfinunfprcdwzb&nodeId=phkfinunfprcdwzb-initialprimary#L4537) on OL-8.5:
```
Error from server (InternalError): error when creating "./kustomize/minio/": Internal error occurred: failed calling webhook "admission-webhook.openebs.io": Post "https://admission-server-svc.openebs.svc:443/validate?timeout=5s": dial tcp 10.96.2.96:443: connect: connection refused
```

**Pull request closed without merging**